hypervisor/debug/uart16550.c

```diff
@@ -14,20 +14,28 @@
 
 #define MAX_BDF_LEN 8
 
+struct uart_addr {
```

Comment:
we already have definition of vuart_addr, it is too close.
struct console_uart ?

Reply:
OK. Could I add other fields (like uart_enabled, uart_rx_lock etc) to like structure too ?